routes/discord.js

```diff
@@ -18,4 +23,54 @@ router.post('/generate-invite', verifyRole(ROLE_NAMES.member), async (req, res)
   } 
 });
 
+router.get('/auth/discord', (req, res) => {
+  res.redirect(`https://discord.com/api/oauth2/authorize?client_id=${discordAuth.discordClientId}
+    &redirect_uri=${encodeURIComponent(discordAuth.discordRedirectUrl)}&response_type=code&scope=identify%20email`);
+});
+
+router.get('/auth/discord/callback', async (req, res) => {
```

Comment:
as discussed yesterday, let's create a ticket for Frontend so they create a callback endpoint on their side + send us the code they receive from Discord. and get rid of this endpoint here.